Sources/SkipWeb/WebEngine.swift

```diff
@@ -493,19 +506,19 @@ public struct WebLoadError : Error, CustomStringConvertible {
 
 
 /// The configuration for a WebEngine
-public class WebEngineConfiguration : ObservableObject {
-    @Published public var javaScriptEnabled: Bool
-    @Published public var allowsBackForwardNavigationGestures: Bool
-    @Published public var allowsPullToRefresh: Bool
-    @Published public var allowsInlineMediaPlayback: Bool
-    @Published public var dataDetectorsEnabled: Bool
-    @Published public var isScrollEnabled: Bool
-    @Published public var pageZoom: CGFloat
-    @Published public var isOpaque: Bool
-    @Published public var customUserAgent: String?
-    @Published public var userScripts: [WebViewUserScript]
-    @Published public var messageHandlers: [String: ((WebViewMessage) async -> Void)]
-    @Published public var schemeHandlers: [String: URLSchemeHandler]
+@Observable public class WebEngineConfiguration {
```

Comment:
@dfabulich This will technically be a breaking change to the SkipWeb API. Will this cause problems for you if we change WebEngineConfiguration from `ObservableObject` to `@Observable`?

Reply:
Nah, go for it